SPM Example/PortalSwift/ViewController.swift

```diff
@@ -220,29 +230,151 @@ class ViewController: UIViewController, UITextFieldDelegate {
       throw PortalExampleAppError.userNotLoggedIn()
     }
     guard let config else {
-      self.logger.error("ViewController.recover() - ❌ Application configuration not set.")
+      self.logger.error("ViewController.eject() - ❌ Application configuration not set.")
       throw PortalExampleAppError.configurationNotSet()
     }
-    guard let cipherTextUrl = URL(
-      string: "\(config.custodianServerUrl)/mobile/\(user.exchangeUserId)/cipher-text/fetch?backupMethod=\(withBackupMethod.rawValue)"
-    ) else {
-      throw URLError(.badURL)
+
+    var cipherText: String? = nil
+    var organizationShare: String? = nil
```

Comment:
```suggestion
    var cipherText: String?
    var organizationShare: String?
```
Unnecessary Initialization